ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/api/data_api.py

```diff
@@ -33,6 +33,630 @@ class DataApi(ApiBase):
     Ref: https://github.com/swagger-api/swagger-codegen
     """
 
+    def export_binary_data_for_latest(
+        self,
+        *,
+        database_key: "str",
+        record_history_guid: "str",
+        attribute_guid: "str",
+        mode: "Optional[str]" = None,
+        x_ansys_vc_mode: "Optional[str]" = None,
+        max_width: "Optional[int]" = None,
+        max_height: "Optional[int]" = None,
+    ) -> "Union[None, str]":
+        """Get the binary data for a given attribute and the latest version of a record history in given database. Must be a file or picture attribute.
```

Comment:
The second sentence needs a subject for clarity. What must be a file or picture attribute?

Reply:
The content of these strings isn't something we can control, it comes directly from the REST API definition in the Granta MI backend. Over time they will be reviewed, but it's certainly not something that we'll be able to resolve this release

Reply:
I would also add that use of this package is unsupported and we advise people to use the Pythonic packages instead. We'll always make sure that these more Pythonic libraries built on top of this have properly-reviewed docstrings,